### DISCUSSION TITLE
[Suggestion] Add command for other [ all ] players to use villagers

### DISCUSSION THREAD

**Discussion by iTaeyeonKim** [GriefPrevention/GriefPrevention#251] [Q&A] [answered]
Adding permission for [ all ] also can 1 player to use villagers, without giving access to kill them

**Comment by TheSenseiNL** [comment]
This isnt really an issue tbh.

Just make a sub claim with /subdivideclaim
Then in that subdivided claim do:  /containertrust ALL

That gives everyone access to traid with the villagers. I use that to. Put the villagers in between half slabs or anything. They can trade but people can't kill them.

**Comment by RoboMWM** [comment, marked as answer]
Or just `/containertrust public` since it sounds like you're trying to apply this to a claim anyways.

Also I recommend using `public` instead of `all` since doing `/untrust all` removes everyone from the trustlist instead of just removing the `public` from it.

Thanks for the tip TheSenseiNL
